### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chrismwsas**
I've reorganised the README to move the focus from deployment with UAA to usage in Azure Marketplace. 

I've said that only internal SAS users should deploy the plugin with UAA.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,76 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
 
 <img alt="Sailing dashboard" src="img/sailing-dashboard.png"  width="50%" height="50%">
 
-## Prerequisites
+
+## Getting Started
+The following steps provide an example of how to get started with the plug-in. 
+
+### Prerequisites
+Have a running ESP Marketplace deploy.
```
Comment (katjam-sas) [inline README.md]: There are 2 prereqs here so how about a bulleted list, as shown below. Notice also how I have referred to Marketplace as "SAS Event Stream Processing in the Microsoft Azure Marketplace":

- A running deployment of SAS Event Stream Processing in the Microsoft Azure Marketplace.
- An ESP project running in either SAS Event Stream Processing Studio or SAS Event Stream Manager.

File: README.md [hunk 2]
```diff
@@ -9,15 +9,76 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
...
+## License
+
+> This project is licensed under the [Apache 2.0 License](LICENSE).
+
+## Additional Resources
+- [Grafana documentation](https://grafana.com/docs/)
+- [Grafana tutorials](https://grafana.com/tutorials/)
+
+## SAS Internal Deployment Notes
+This section is only relevant to user working for SAS.
```
Comment (katjam-sas) [inline README.md]: How about:
"This section is relevant only to internal users at SAS."

File: README.md (hunk 1 above)
Comment (chrismwsas) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (chrismwsas) [reply README.md]: Done

**Pull request merged**